extension/content.js

```diff
@@ -344,6 +344,22 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addUploadBtn() {
+	if (!pageDetect.isPRFiles()) {
```

Comment:
I agree with @sindresorhus's point. We should be checking for a comment box in the DOM.

To do that this can be replaced with a function to check if there is a comment box similar to `hasCode()`:

```
const hasComments = () => isRepo() && $('.js-previewable-comment-form').length > 0;
```